back/spec/acceptance/users_spec.rb

```diff
@@ -23,11 +23,22 @@
         parameter :size, 'Number of users per page'
       end
       parameter :search, 'Filter by searching in first_name, last_name and email', required: false
-      parameter :sort, "Sort user by 'created_at', '-created_at', 'last_name', '-last_name', 'email', '-email', 'role', '-role'", required: false
+      parameter :sort, "Sort user by 'created_at', '-created_at', 'last_name', '-last_name', 'email', " \
+                       "'-email', 'role', '-role'", required: false
       parameter :group, 'Filter by group_id', required: false
-      parameter :can_moderate_project, 'Filter by users (and admins) who can moderate the project (by id)', required: false
-      parameter :can_moderate, 'Filter out admins and moderators', required: false
-
+      parameter :can_moderate, 'All admins + users with either a project &/or folder moderator role', required: false
+      parameter :can_moderate_project, 'All admins + users who can moderate the project (by project id), ' \
+                                       'excluding folder moderators of folder containing project ' \
+                                       '(who can, in fact, moderate the project), ' \
+                                       'OR Admins & all users with project moderator role ' \
+                                       '(if no project ID provided)', required: false
```

Comment:
@brentguf Maybe the behaviour 'excluding folder moderators of folder containing project' is not the desired behaviour?

I am documenting the current behaviour, however, to help you make such a decision.

See [related comment](https://github.com/CitizenLabDotCo/citizenlab/pull/4455/files#r1153347164) in model spec.